types/markdown-it-link-attributes/index.d.ts

```diff
@@ -1,18 +1,26 @@
 // Type definitions for markdown-it-link-attributes 3.0
 // Project: https://github.com/crookedneighbor/markdown-it-link-attributes
 // Definitions by: Katy Richard <https://github.com/cowpewter>
+//                 Piotr Błażejewicz <https://github.com/cowpewter>
```

Comment:
Looks like you forgot to fix your own github username!

Reply:
oh, embarassng, isn't it?

Reply:
amended, sorry for being a squatter :)